packages/components/script.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from "react";
+import { connect } from "frontity";
+import { Connect, Package } from "frontity/types";
+
+type ScriptType = React.FC<
+  Connect<Package, { src?: string; code?: string; dataId?: string }>
+>;
+
+const Script: ScriptType = ({ state, ...props }) => {
+  let script = null;
+  const { src, code, dataId } = props;
+
+  useEffect(() => {
+    if (state.frontity.rendering === "csr") {
```

Comment:
I think there's not much sense to use `state.frontity.rendering` now that we are returning null.